cpp/src/ice2slice/Gen.cpp

```diff
@@ -649,7 +653,26 @@ Gen::TypesVisitor::visitEnum(const EnumPtr& p)
     }
     out.dec();
     out << nl << "}";
-    out << nl;
+}
+
+void
+Slice::Gen::TypesVisitor::visitConst(const ConstPtr& p)
+{
+    Output& out = getOutput(p);
+    out << sp;
+    out << nl << "// ice2slice could not convert:";
+    out << nl << "// const " << p->name() << " = " << p->value();
```

Comment:
Would be nicer if we can get the original definition, for the warning:

```
// ice2slice could not convert:
// const int PermissionWhisper = 256
```

Was generated from:

```
const int PermissionWhisper = 0x100;
```

Reply:
There's no way to do this at the moment, but feel free to open an issue, and I can add a field that stores the original raw text, so we can add it to this comment.